src/Relation/Binary/Construct/Interior/Symmetric.agda

```diff
@@ -56,28 +59,59 @@ transitive tr = trans tr tr
 asymmetric⇒empty : Asymmetric R → Empty (SymInterior R)
 asymmetric⇒empty asym (r , r′) = asym r r′
 
+-- Decidability
+decidable : Decidable R → Decidable (SymInterior R)
+does  (decidable R? x y) = does (R? x y) ∧ does (R? y x)
+proof (decidable R? x y) = proof (R? x y) reflects proof (R? y x)
+  where
+  _reflects_ : ∀ {bxy byx} → Reflects (R x y) bxy → Reflects (R y x) byx →
```

Comment:
Is it worth pulling this to the top level?

Reply:
That's a nice question!
My first thought was that this was a fairly slavish re-emulation of the corresponding (top-level) principle `_×-reflects_` defined in `Relation.Nullary.Reflects`, so didn't merit elevation to the top-level; not least because it really only has this one use, namely to establish decidability... (in that sense, almost *none* of `Reflects` should really be exposed outside `Relation.Nullary.Decidable.*`?).
Not sure how best to reply except that: if we think we might use it later/downstream, we could always refactor to lift it back out...? But going the other way is ... more work ;-)
Not the most 'dignified' answer!

Reply:
I'm with @Taneb here, this does seem to be of independent interest... sorry!

Reply:
@JacquesCarette comments:
> I'm with @Taneb here, this does seem to be of independent interest... sorry!

I think that if you the two of you can come up with a good name for this lemma, I'll do the refactoring...
... else let's merge this now against such a name choice emerging later...?